docs/interpretation.rst

```diff
@@ -150,7 +150,7 @@ with the hypothetical gene YFG42.
         id: "OMIM:263750"
         label: "Miller syndrome"
       genomicInterpretations:
-      - interpretationStatus: "CONTRIBUTORY"
+      - interpretationStatus: "CANDIDATE"
```

Comment:
In this case the interpretationStatus should be CAUSAL, this was a published case!

Reply:
Maybe it does not fit well with the description, but it si not wrong, I will merge